apps/web/src/views/Landing/Why.tsx

```diff
@@ -0,0 +1,54 @@
+import { Heading, VStack, Box } from '@chakra-ui/react';
+import React from 'react';
+import { Chat } from '../../components';
+
+const Why = () => {
+    const chats = [
+        {
```

Comment:
since this is a constant array place it outside the component otherwise array will rerender everytime when component rerenders "performance issue"

Reply:
fixed in [3786147](https://github.com/tinkerhub/saturday-hack-night/pull/8/commits/3786147dcca1c29389a2f084c8cc1b1138b4d408)